geomesa-fs/geomesa-fs-spark-runtime/src/test/scala/org/locationtech/geomesa/fs/spark/FileSystemRDDProviderTest.scala

```diff
@@ -106,7 +106,7 @@ class FileSystemRDDProviderTest extends Specification with LazyLogging {
             "dtg > '2016-01-01T12:00:00Z' AND dtg < '2016-01-02T12:00:00Z'"
         val rows = sc.sql(select).collect()
         rows must haveLength(1)
-        rows.head.get(0) mustEqual "2"
+        rows.head.get(rows.head.fieldIndex("__fid__")) mustEqual "2"
```

Comment:
this type of thing makes me think maybe this change should require a major version bump, or maybe an opt-in flag or something. or we could just add it to upgrade guide...